extensions/integration-tests/src/index.ts

```diff
@@ -4,27 +4,33 @@
  *--------------------------------------------------------------------------------------------*/
 import * as vscode from 'vscode';
 import { context } from './testContext';
+import { getSuiteType, SuiteType } from './fmkUtils';
 
 const path = require('path');
 const testRunner = require('vscode/lib/testrunner');
 
-const suite = 'Integration Tests';
-
 const options: any = {
 	ui: 'tdd',
 	useColors: true,
-	timeout: 600000
+	timeout: 600000 	// 600 seconds
 };
 
+const suite = getSuiteType();
```

Comment:
This removes the current default of suite = 'Integration' when environment variable is not default. This default should be restored.

Reply:
resolved.